src/Pages/PassengerInfo/Accordian.js

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+function Accordian({ title }) {
+  const content = useRef(null);
+  const [height, setHeight] = useState(0);
+  const toggleAccordion = () => {
+    setHeight(height === 0 ? content.current.scrollHeight + 30 : 0);
+  };
+  const checkInfo = () => {};
+
+  return (
+    <AccordianAll>
+      <AccordionHeading>
+        <Container>
+          <TitleStyle>{title}</TitleStyle>
+          <button onClick={toggleAccordion}>Click</button>
+        </Container>
+      </AccordionHeading>
+
+      <AccrodionContent height={height} ref={content}>
+        <Container1>
```

Comment:
```suggestion
        <InputWrapper>
```
변수명에 1, 2, 3 등 숫자를 넣는건 별로 좋지 네이밍인 것 같습니다. 위와 같은 이름은 어떨까요?